src/elements/ItemsContainerElement.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from 'react';
+
+const createElement = ({ id, className }: IProps) => ({
+    __html: `<div
+        is="emby-itemscontainer"
+        ${id}
+        class="${className}"
+    >
+    </div>`
+});
+
+type IProps = {
```

Comment:
prefer interface over type for these kind of typings.

Reply:
I'm still pretty new to Typescript. Why would an interface be preferred here? Most of the information I can find says it doesn't really matter, but types might be preferable since they support some things interfaces do not.

Reply:
well interfaces resemble objects/classes/components. The way i see it, when you have components or objects, you can usually represent using a interface. They can also be extended, inheriting the super interface properties.

When you want to represent some helper, like Record, or array or literal strings, you need to use types. In some cases, a interface signature requires types to function property in conjunction with generics, but that's a little more advanced ahah.

I am no expert, but i normally use it that way.